src/MissionManager/MavCmdInfoCommon.json

```diff
@@ -1112,6 +1112,20 @@
                 "enumValues":       "3,4"
             }
         },
+        {
+            "id":                   42600,
+            "rawName":              "MAV_CMD_DO_WINCH",
+            "friendlyName":         "Perform winch action",
+            "description":          "Perform winch action.",
+            "category":             "Winch",
+            "friendlyEdit":         true,
+            "param2": {
+                "label":            "Winch action",
+                "default":          4,
+                "enumStrings":      "Drop,Retract",
+                "enumValues":       "4,6"
```

Comment:
Missing relax/release with id 0?

Reply:
I decided not to include it, because planning an emergency release seemed weird to me. I feared that the pilot might select the wrong option by accident. I can add it if this is something we want.